reco_utils/common/gpu_utils.py

```diff
@@ -23,6 +24,40 @@ def get_number_gpus():
         return 0
 
 
+def get_gpu_info():
```

Comment:
I would recommend to use the numba api, it is much faster than nvidia-smi. Here you have an example of what you want: https://github.com/miguelgfierro/codebase/blob/master/python/system/memory.py#L59